### PR TITLE
Gate Seal scenario tests

### PR DESCRIPTION
**Pull request by Jeday**
# Checked
- gate seal config
- withdrawals work
- enact gateseal (stop withdrawals or stop exit queue bus)
- finalization and new requests disabled anything else work
- expires after year

### CODE REVIEW

File: tests/regression/test_gate_seal.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+from utils.config import (
+    contracts,
+    deployer_eoa,
+    lido_dao_withdrawal_queue,
+    lido_dao_validators_exit_bus_oracle,
+    gate_seal_address,
+)
+
+
+DURATION_SECONDS = 6 * 24 * 60 * 60  # 6 days
```
Comment (TheDZhon) [inline tests/regression/test_gate_seal.py:16]: maybe `PAUSE_DURATION_SECONDS`

File: tests/regression/test_gate_seal.py [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    assert contracts.gate_seal.get_seal_duration_seconds() == DURATION_SECONDS
+    sealables = contracts.gate_seal.get_sealables()
+    assert len(sealables) == 2
+    assert lido_dao_withdrawal_queue in sealables
+    assert lido_dao_validators_exit_bus_oracle in sealables
+    assert contracts.gate_seal.get_expiry_timestamp() == EXPIRY_TIMESTAMP
+    assert not contracts.gate_seal.is_expired()
+
+
+def test_gate_expiration():
```
Comment (TheDZhon) [inline tests/regression/test_gate_seal.py:36]: maybe `test_gate_expiration` → `test_gate_seal_expiration`

File: tests/regression/test_gate_seal.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    lastCheckpointIndex = contracts.withdrawal_queue.getLastCheckpointIndex()
+    hints = contracts.withdrawal_queue.findCheckpointHints(pending_request_ids, 1, lastCheckpointIndex)
+    claim_balance_before = account.balance()
+    claim_tx = contracts.withdrawal_queue.claimWithdrawals(pending_request_ids, hints, {"from": steth_holder})
+    claim_balance_after = account.balance()
+
+    assert almostEqEth(
+        claim_balance_after - claim_balance_before + claim_tx.gas_used * claim_tx.gas_price, REQUESTS_SUM
+    )
+    assert claim_tx.events.count("WithdrawalClaimed") == REQUESTS_COUNT
```
Comment (TheDZhon) [inline tests/regression/test_gate_seal.py:166]: I would show that both contracts are still paused (like a cherry on a cake)
i.e., highlight that the claim hasn't been prohibited by GateSeal

**Review by TheDZhon — APPROVED** [review]
LGTM :sunglasses: 
Perhaps, only a few minor things to pick up

**Review by bulbozaur — APPROVED** [review]
LGTM

**Pull request merged**